File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.java
// Last updated: 7/16/2025, 9:42:15 AM
class Solution {
    public int[] runningSum(int[] nums) {
        for (int i = 1; i < nums.length; i++){
            nums[i]= nums[i-1]+nums[i];
        }
        return nums;
    }
}

File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.py
# Last updated: 7/16/2025, 9:42:10 AM
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        curr = 0
        ans = []
        for num in nums:
            ans.append(num + curr)
            curr += num
        return ans